src/store/accountStore.js

```diff
@@ -196,7 +191,10 @@ export default {
     wallets: state => {
       var wallets = []
       if (state.accounts) {
-        // window.accounts = state.accounts
+        if (!window.accounts) {
+          window.accounts = state.accounts
```

Comment:
Can this be put in some store?

Reply:
code for testing purposes, can be removed